### PR TITLE
Golem.de: add more secured subdomains

### PR DESCRIPTION
**Pull request by numismatika**
- [x] check the damoh ad subdomains for a possible replacement with a wildcard

### CODE REVIEW

**Comment by rid007** [comment]
Golem.de just activate /added https to main page. 
With HTTPS Everywhere Block all unencrypted requests works perfect. 
But still not server site HSTS support.

Please add in the golem.de xml `https://golem.de, https://www.golem.de & https://video.golem.de`

**Comment by numismatika** [comment]
both www and video are in, just needs a merge imho

**Comment by numismatika** [comment]
Golem now made TLS support official.
@jeremyn  could we merge it now?

File: src/chrome/content/rules/Golem.de.xml [hunk 1]
```diff
@@ -1,23 +1,44 @@
...
+	<target host="sidebar.golem.de" />
 	<target host="suche.golem.de" />
+	<target host="vcs1.golem.de" />
+	<target host="video.golem.de" />
+	<target host="wap.golem.de" />
+	<target host="www6.golem.de" />
+
+	<test url="http://0b2792c6455becc24114e642f0b276c68f9d0862.1.damoh.golem.de/" />
+	<test url="http://019a1d8d59674aefda20c1057486ffeb0e066806.2.damoh.golem.de/" />
+	<test url="http://0fe93d630211716523fce0dac877fdb2a23f1f896.3.damoh.golem.de/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Golem.de.xml]: We require 3 test urls for each wildcard host, but these are just one by one.

**Comment by numismatika** [comment]
Rebased and extended with a few more urls

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**